apps/explorer/src/app/app.tsx

```diff
@@ -1,18 +1,16 @@
 import classnames from 'classnames';
-import { useState, useEffect } from 'react';
-import { useLocation } from 'react-router-dom';
 import { EnvironmentProvider, NetworkLoader } from '@vegaprotocol/environment';
 import { Nav } from './components/nav';
 import { Header } from './components/header';
 import { Main } from './components/main';
 import { TendermintWebsocketProvider } from './contexts/websocket/tendermint-websocket-provider';
-import type { InMemoryCacheConfig } from '@apollo/client';
 import { Footer } from './components/footer/footer';
 import { AnnouncementBanner, ExternalLink } from '@vegaprotocol/ui-toolkit';
 import {
   AssetDetailsDialog,
   useAssetDetailsDialogStore,
 } from '@vegaprotocol/assets';
+import { DEFAULT_CACHE_CONFIG } from '@vegaprotocol/apollo-client';
```

Comment:
**praise (non-blocking)**: Thanks for making this happen. It's good to share, and obviously the defaults are still extendable. Excellent!